golang/dashboard/src/components/Relay-Info/Index.vue

```diff
@@ -0,0 +1,80 @@
+<template>
```

Comment:
Rename this file to `Relay-Data`, `Relay-Main`, or something similar. Avoid index.vue names because they are non-descript.